src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs

```diff
@@ -19,13 +19,8 @@ public class PushToBlobFeed : MSBuild.Task
         [Required]
         public string AccountKey { get; set; }
 
-        [Required]
         public ITaskItem[] ItemsToPush { get; set; }
 
-        public string IndexDirectory { get; set; }
```

Comment:
Does Sleet use any temp directories that we need to override the location for? If so we should still have a version of this property to repoint those.

Reply:
Sleet uses its own internal temp directories but there is no way to override the directory